### PR TITLE
Updating team members

### PR DESCRIPTION
**Pull request by ProLoser**
Added myself. Need other people to add / update themselves and suggest removals. Can be done on this PR or in a new one.

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -9,16 +9,20 @@
...
   gravatar: 15a22b93c94ecd7c8cf880d8c5ac18ca
   email: emily@codeforsanfrancisco.org
 - name: Jason Lally
-  title: Lead Organizer
+  title: Organizer
   gravatar: b0d79f25e915d34f7f06907322ad8060
   email: jason@codeforsanfrancisco.org
 - name: Jesse Szwedko
-  title: Lead Organizer
+  title: Organizer
```
Comment (jszwedko) [inline _data/team.yml]: :+1: you can drop me from this page since I'm not active day-to-day.

Comment (ProLoser) [reply _data/team.yml]:
```suggestion
  title: Web Admin
```

How about this since current assistance has been appreciated?

Comment (jszwedko) [reply _data/team.yml]: :smile: I appreciate the thought, but I still think it'd be more accurate for this to just reflect the current core team. I'm, of course, still happy to help through the transitions.

File: _data/team.yml [hunk 2]
```diff
@@ -2,25 +2,22 @@
...
   gravatar: 900717b6b0d5430cbfb953d44cacd1de
-- name: Anthony Faber
-  title: Door Wrangler
-  gravatar: 15da69259d301b81d09f16a8ef342d62
-- name: Chuck Temple
-  title: Project Investment Fund Lead
-  gravatar: fd6fd28a3a89cd5416d47ca691a08442
-  email: chuck@codeforsanfrancisco.org
+- name: Dean Sofer
+  title: Project Success
```
Comment (emily-wasserman) [inline _data/team.yml]: I don't think this is going to be informative for regular brigade members. Can you think of another way to communicate what you mean?

File: _data/team.yml [hunk 3]
```diff
@@ -2,25 +2,22 @@
...
+  gravatar: ae341f7eb365518d32b1b6bec287fa8b
+  email: dean@codeforsanfrancisco.org
 - name: Emily Wasserman
-  title: Organizer
+  title: Brigade Captain
+  gravatar: 15a22b93c94ecd7c8cf880d8c5ac18ca
+  email: emily@codeforsanfrancisco.org
+- name: Gregory Boyer
+  title: Finance
```
Comment (emily-wasserman) [inline _data/team.yml]:
```suggestion
  title: Finance Lead, Operations Coordinator
```

Comment (emily-wasserman) [reply _data/team.yml]: Let's make sure to represent the full extent of what people do

File: _data/team.yml [hunk 4]
```diff
@@ -2,25 +2,22 @@
...
   gravatar: 15a22b93c94ecd7c8cf880d8c5ac18ca
   email: emily@codeforsanfrancisco.org
 - name: Jason Lally
-  title: Lead Organizer
+  title: Organizer
```
Comment (emily-wasserman) [inline _data/team.yml]:
```suggestion
  title: Organizer, Open Data Liaison
```

File: _data/team.yml [hunk 5]
```diff
@@ -2,25 +2,22 @@
 - name: Allen Meyers
   title: Storyteller
```
Comment (emily-wasserman) [inline _data/team.yml]:
```suggestion
  title: Storyteller, Partnership Liaison
```

File: _data/team.yml [hunk 6]
```diff
@@ -2,25 +2,22 @@
...
   email: jason@codeforsanfrancisco.org
-- name: Jesse Szwedko
-  title: Lead Organizer
-  gravatar: 3e3a63b9ce04e1cd241d8660248e5412
-  email: jesse.szwedko@codeforsanfrancisco.org
-- name: Peggy Lee
-  title: Hack Night Coordinator
-  gravatar: af22c106c5c1154389bb4329fc15fb21
+- name: Reid Sherman
+  title: Advocate
```
Comment (emily-wasserman) [inline _data/team.yml]:
```suggestion
  title: Brigade Member Ambassador
```

Comment (ProLoser) [reply _data/team.yml]: @reidjs do you have a gravatar we can use?

File: _data/team.yml (hunk 2 above)
Comment (ProLoser) [reply _data/team.yml]: Ummm... yeah I was trying to think of something better.

Product Manager Consultant?
Project Support?

Comment (emily-wasserman) [reply _data/team.yml]: Project Success Consultant, maybe? Up to you!

**Review by emily-wasserman — APPROVED** [review]
Looks great, let's merge it

**Pull request merged**